server/src/test/java/com/cloud/storage/VolumeApiServiceImplTest.java

```diff
@@ -580,9 +580,86 @@ public void validateConditionsToReplaceDiskOfferingOfVolumeTestEverythingWorking
         inOrder.verify(storagePoolMock).isLocal();
         inOrder.verify(newDiskOfferingMock, times(0)).isShared();
         inOrder.verify(volumeApiServiceImpl).getStoragePoolTags(storagePoolMock);
-        inOrder.verify(newDiskOfferingMock).getTags();
 
         inOrder.verify(volumeVOMock).getSize();
         inOrder.verify(newDiskOfferingMock).getDiskSize();
     }
+
+    @Test
+    public void doesTargetStorageSupportNewDiskOfferingTestDiskOfferingMoreTagsThanStorageTags() {
```

Comment:
wow, how java of you @rafaelweingartner , can you translate this name?

Reply:
Sure, this is something I do to control the unit tests. The pattern I normally use is the following:
<methodNamte>Test<parameterStateBeingTested>
Translating this case:
I am testing the method `doesTargetStorageSupportNewDiskOffering` with the following condition:
* the disk offering will have more tags than the target storage

There is also another format I use:
`<methodNamte>Test`
Happy day test. It can be the case when the method finishes successfully or a method that does not have different execution flows.